### ISSUE TITLE
Please stop merging PRs involving strings without l10n review

### ISSUE THREAD

**Issue by flodolo** [mozilla/testpilot#2574]
@clouserw @johngruen 
I've tried multiple times to raise this issue, and failed. Let's try with a public issue, because I'm running out of options.

Here's what happened last night. This landed
https://github.com/mozilla/testpilot/commit/04c94f896f11f85f13748f9e65921f4e4a8cff24

And then this landed
https://github.com/mozilla/testpilot/commit/9f228465c97e7fb79804907dd9a1f5b4f84d4583

This morning I wake up, find suggestions on Pontoon from one of my fellow translators (@gialloporpora), and they make no sense, because they don't match the English text. So I go in, check the history, and find this mess.

Now we need to:
* Change the string ID for those 3 messages, to make sure people translate the right content.
* Send a message out to explain what happened.

I think the only way out of this is to make sure any string change lands reviewed beforehand by me, and that's not great, because Test Pilot is not the only thing I'm following, and I'm 9 hours away from most of the devs.

If you have better ideas, I'm all hears.

**Comment by lmorchard** [comment]
We haven't gotten down to executing on this yet, but we have a notion of a plan to change how things work here: https://github.com/mozilla/testpilot/issues/2430#issuecomment-302809115

TL;DR: Move any changes involving strings to an l10n branch. Make PRs into that branch for l10n review. Once strings are localized, make a PR from l10n to master to land the feature. What we might be missing, though, is who else to ping for l10n reviews so we're not always pestering you (@flodolo)

I think this week has been a bit chaotic because we're all at the SF work week and trying to make some things happen quickly

**Comment by johngruen** [comment]
+1 @lmorchard.  We could also look at setting flags in circle CI to warn us if we change values in one of the yaml files so that we make sure to appropriately modify keys.

**Comment by flodolo** [comment]
> TL;DR: Move any changes involving strings to an l10n branch. Make PRs into that branch for l10n review. Once strings are localized, make a PR from l10n to master to land the feature. What we might be missing, though, is who else to ping for l10n reviews so we're not always pestering you (@flodolo)

OK, that would solve a lot of issues. The challenges are:
* To make sure we merge changes frequently from the l10n branch into master
* Figure out a way to have a testing server that has the latest strings translated

**Comment by lmorchard** [comment]
Just as an update for the same of completeness: I just merged in a [CODEOWNERS file](https://github.com/mozilla/testpilot/blob/master/CODEOWNERS) to automatically request review when `locales/en-US` files change. 

But, I think that's just an initial step - and a github feature that didn't exist a few weeks ago. We should still continue forward with a new l10n branch and testing server ASAP (i.e. probably when I'm done with #2423)

**Comment by ghost** [comment]
Les is going to work on this once he finishes the add-on conversion

**Comment by fzzzy** [comment]
We have the new l10n branch flow in place. Localizers will always have a week to localize strings. I think this can be closed now.
